src/Sylius/Bundle/PayumBundle/Provider/PaymentDescriptionProviderInterface.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sylius\Bundle\PayumBundle\Provider;
+
+use Sylius\Component\Core\Model\PaymentInterface as SyliusPaymentInterface;
```

Comment:
It was there as probably some other `PaymentInterface` has been used. It obviously should be deleted ;)